ops/ccs-ops-misc/cert-checker/cert-checker.sh

```diff
@@ -0,0 +1,245 @@
+#!/usr/bin/env bash
+# Check/validate TLS endpoints and certificate files and warn of upcoming expirations
+PROGNAME=${0##*/}
+
+# whether or not to send out notifications (defaults to false, add -s flag to enable)
+SEND_NOTIFICATIONS=${SEND_NOTIFICATIONS:-"false"}
+
+# schedule info
+CRON_SCHEDULE="${SCHEDULE:-"0 12 * * *"}" # run at noon everyday
+
+# notifications
+WARN_WHEN="${WARN_WHEN:-"3000:4500"}"     # WARN when expiration is >= 30 but less than 45 days
+ALERT_WHEN="${ALERT_WHEN:-"1500:3000"}"   # expiration is >= 15 but less than 30 days
+PAGE_WHEN="${PAGE_WHEN:-"1:1500"}"        # expiration is >= 0 but less than 15 days
```

Comment:
these values are very different from the sample which are '30:45', etc. Which is correct?

Reply:
changed